src/antelope/chains/NativeChainSettings.ts

```diff
@@ -123,6 +123,13 @@ export default abstract class NativeChainSettings implements ChainSettings {
     abstract getTheme(): Theme;
     abstract getFiltersSupported(prop: string): boolean;
 
+    // new methods
```

Comment:
I would suggest a comment indicating what are tokens that are considered important

Reply:
great idea!